### PR TITLE
Define working groups

### PR DESCRIPTION
**Pull request by malomarrec**
Having a working group structure helps deliver and communicate about cross-functional initiatives. They've been emerging organically in engineering, product and marketing initiatives. This brings minimal documentation on what working groups are, when to create one, and how to communicate.

### CODE REVIEW

File: handbook/product/working-group.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+## Ending a working group
+
+After the outcome has been achieved, or after a given time period has elapsed, the working group should end. If the working group lives for too long (more than two quarters), it probably means that the outcome was too big and needs to be scoped down, or that responsibilities should be transferred to a team.
+
+When you end the working group
+- run a [retrospective](../retrospectives.md) if it's useful
+- write and share a wrap up, share the final outcome
+- archive the slack channel
```
Comment (dadlerj) [inline handbook/product/working-group.md:46]:
```suggestion
- archive the Slack channel
```

**Review by dadlerj — APPROVED** [review]
I think this is a great addition. Since it's still so early I imagine this will evolve a lot in the next few months, and we should be sure to come back and refine this as we learn what works and what doesn't. 

The only serious comment I have is the one about moving this to a company-wide part of the handbook. 

Thanks for adding!

File: handbook/product/working-group.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Using working groups to deliver cross-team initiatives
```
[the remark this answers is not in the document]
Comment (malomarrec) [reply handbook/product/working-group.md]: Thanks for all the feedback. Moved to communication!

**Review by felixfbecker — APPROVED** [review]
This is great! I think it will support more cross-team collaboration :)

**Pull request merged**